tests/pipelines/pixart_sigma/test_pixart.py

```diff
@@ -336,23 +336,28 @@ def test_pixart_1024(self):
         image = pipe(prompt, generator=generator, num_inference_steps=2, output_type="np").images
 
         image_slice = image[0, -3:, -3:, -1]
-        expected_slice = np.array([0.0742, 0.0835, 0.2114, 0.0295, 0.0784, 0.2361, 0.1738, 0.2251, 0.3589])
+        expected_slice = np.array([0.4517, 0.4446, 0.4375, 0.449, 0.4399, 0.4365, 0.4583, 0.4629, 0.4473])
```

Comment:
Pretty sure these obtained on a different hardware.

Reply:
I got the expected result on V100 GPU. But, I remember I passed the text before.

Reply:
Our runner is based on t4. But not a big problem.

Reply:
Got it.